modules/puzzles/util.py

```diff
@@ -52,7 +53,7 @@ def locations_containing_symbols(symbols, ls=locations()):
     s_dict = Counter(symbols.lower())
 
     for symbol in s_dict:
-        ls = [l for l in ls if l.name.lower().count(symbol) >= s_dict[symbol]]
+        ls = [l for l in ls if l.name.lower().replace('ë','e').replace('é','e').count(symbol) >= s_dict[symbol]]
```

Comment:
Ik zou de van `l.name.lower().replace('ë','e').replace('é','e')` een variabele maken. Dat hoeft python dit niet telkens opnieuw uit te rekenen en is het iets leesbaarder.

Reply:
Kan ik dat in de list comprehension doen of vervang ik die dan voor een for loop?

Reply:
Oh, oops, had neit gezien dat dit op `l` gebeurd die in de list comprehensie zit. Ik zou in dat geval misschien een util functie maken die een 'canonical' string maakt.